tests/unit/backend/api/controllers/PullRequestController.spec.js

```diff
@@ -0,0 +1,48 @@
+import PullRequestController from '../../../../backend/api/controllers/PullRequestController'
+import axios from 'axios'
+import mockedResponse from '../../support/response.json'
+import fetchPullRequests from '../../../../backend/workers/fetch-pull-requests'
+import addedData from '../../support/addedData.json'
+import PullRequest from '../../../../backend/services/PullRequest'
```

Comment:
Some paths are wrong, with a missing `../`,
And there are also some variables that are never used.

use `npm run lint` to check for linter errors.